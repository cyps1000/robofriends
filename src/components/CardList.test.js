import React from "react";
import { shallow, mount, render } from "enzyme";
import renderer from "react-test-renderer";
import CardList from "./CardList";

it("expect to render Card Component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "Jon Snow",
      userName: "jonJon",
      email: "jon@nightwatch.com",
    },
  ];
  const tree = renderer.create(<CardList robots={mockRobots} />).toJSON();
  expect(tree).toMatchSnapshot();
});
