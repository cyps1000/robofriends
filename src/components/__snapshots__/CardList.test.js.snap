// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card Component 1`] = `
<div>
  <div
    className="tc bg-silver dib br3 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        Jon Snow
      </h2>
      <p>
        jon@nightwatch.com
      </p>
    </div>
  </div>
</div>
`;
